### PR TITLE
link to static assets guide from app.arc static reference

### PR DESCRIPTION
**Pull request by tbeseda**
Addresses #248 by linking from app.arc > @static to the frontend guide on Static Assets. Also clarifies that the `/public` directory lives at the project root.

### CODE REVIEW

File: src/views/docs/en/reference/app.arc/static.md [hunk 1]
```diff
@@ -99,6 +98,8 @@ static:
 </div>
 </arc-viewer>
 
+> 🔖 The [Frontend Static assets guide](/docs/en/guides/frontend/static-assets) has more information on how to use static assets in your Architect project.
```
Comment (macdonst) [inline src/views/docs/en/reference/app.arc/static.md:101]: Should we have a space between the bookmark emoji and `The`? Just thinking a screen reader may really mangle that pronunciation.

**Review by macdonst — APPROVED** [review]
LGTM with one pedantic comment.

File: src/views/docs/en/reference/app.arc/static.md (hunk 1 above)
Comment (tbeseda) [reply src/views/docs/en/reference/app.arc/static.md:101]: There is a space character. maybe GitHub doesn't kern it well in their code font. Looks like this with the space highlighted
Screen Shot 2021-09-23 at 10 30 15 AM

Maybe a double space?

**Comment by macdonst** [comment]
Huh, I see that there is a space now when I copy and paste the text to an editor. Weirdly on my system it really looks like there is no space.

Screen Shot 2021-09-23 at 12 34 50

Chrome: Version 93.0.4577.82 (Official Build) (arm64)
MacOS: 11.6.0

**Comment by tbeseda** [comment]
> Screen Shot 2021-09-23 at 12 34 50

Ah interesting. I see that a couple of other similar callouts that use a double space between the emoji and the text. I will make a pass and update. Extra whitespace won't hurt Safari/FF

**Pull request merged**